jekyll/_cci2/persist-data.md

```diff
@@ -63,7 +63,7 @@ The information below describes how your network and storage usage is accumulati
 
 **NOTE:** Your overall **Network Transfer** amount is not representative of your billable usage. Only certain actions will result in network egress, which in turn results in billable usage. Details of these actions are described below.
 
-### Overview of storage and network transfer
+### Overview of all storage and network transfer
```

Comment:
```suggestion
### Overview of all storage and network transfers
```

Reply:
Should this be "transfers"?

Reply:
@stalmy what do you think? I also added the "all" to specify that we are not just talking about billable usage, but of both overall N&S as well as billable N&S.